### ISSUE TITLE
Multiple log whitespace is being truncated in the terminal output

### ISSUE THREAD

**Issue by firefixmaarten** [lensapp/lens#2412]
**Describe the bug**
Logs whitespace is being scrambled in terminal output if I force color output using logrus (https://github.com/sirupsen/logrus)

**To Reproduce**
1. use a logrus configuration in the style of:
``` 
// Make sure to log the method
logrus.SetReportCaller(true)

// Some format options
logrus.SetFormatter(&log.TextFormatter{
  ForceColors:            true,
  FullTimestamp:          true,
  DisableLevelTruncation: true,
  PadLevelText:           true,
})
```
2. see the output using kubectl in the terminal
![image](https://user-images.githubusercontent.com/2439070/112963989-30f57e80-9148-11eb-9a6d-a81bc6e2cbe4.png)
3. Compare with the lens buildin terminal
![image](https://user-images.githubusercontent.com/2439070/112964184-613d1d00-9148-11eb-9d77-b7aa5b16a886.png)

**Expected behavior**
The output to retain the original white spacing like in step 2.
Seems like some non asci bytes might cause the issue (this is in vscode after downloading the log), but at least the space chars are all there?
![image](https://user-images.githubusercontent.com/2439070/112965846-f0970000-9149-11eb-948d-8736bbac68d9.png)

**Environment (please complete the following information):**
- Lens Version: 4.1.4
- OS: Fedora 33
- Installation method (e.g. snap or AppImage in Linux): snap

**Comment by Nokel81** [comment]
Question, are the all "space" characters, or could they also be `\t`? I have a sneaking suspicion that logrus (when I used it last) uses both. 

If that is the case, it would explain why INFO lines seem to loose more whitespace.

**Comment by firefixmaarten** [comment]
The downloaded file seems to indicated these are just pure spaces. (ignore the colored green text, it just a style thing)
![image](https://user-images.githubusercontent.com/2439070/113822207-6e2bc300-977d-11eb-8249-a1b0625395aa.png)

As I am now using logfmt style to do processing, I stopped using colors anyway, but I guess the bug could still be fixed for others.

**Comment by firefixmaarten** [comment]
Also, it is also the case in the parts not shown here (without color) after the file etc, multiple spaces are reduced to a single space somehow.
No picture of that anymore, sorry.

**Comment by Nokel81** [comment]
Thanks for the info.

**Comment by Nokel81** [comment]
Fixed by #2655
